app/src/main/res/layout/activity_bmi_calculator.xml

```diff
@@ -14,7 +14,7 @@
         android:textSize="32sp"/>
 
     <ToggleButton
-        android:id="@+id/activity_main_toggleunits"
+        android:id="@+id/toggleunits"
```

Comment:
removed all `activity_main_` prefixes to view ids in this file

Reply:
That makes sense.  That is given by the context.